airgun/views/http_proxy.py

```diff
@@ -58,6 +59,8 @@ class organizations(SatTab):
 
 
 class HTTPProxyEditView(HTTPProxyCreateView):
+    test_button = Text('//a[@id="test_connection_button"]')
```

Comment:
Since you've already defined this in create view with same locator, why repeating it in edit view?